### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by niiku**

### CODE REVIEW

**Comment by incubator-tekton** [comment]
:rocket: Welcome to Baloise ChatOps based GitOps Bot :tada:
                                
    - If you want to enable preview environments for this PullRequest, comment with `/preview`. 
    - If you want to disable the preview environment for this PR, comment with `/undeploy`. 
    - If you need help or want to see the docs, comment with `/help`. 
                                
    Make sure to refresh this page after submitting a command.
    Preview environments can be redeployed automatically if you enable the webhook for updates to the source branch. :new:
    
    Have fun building awesome apps! :hammer:  :whale2:  :truck:  :ship:

**Comment by incubator-tekton** [comment]
New preview environment created for version `5b7906486a400756fba354032a0f67cc79cad46c`. Access it here: https://gitops-nginx-demo-niiku-patch-1-bb5-preview.apps.baloise.dev

File: static/index.html [hunk 1]
```diff
@@ -32,7 +31,7 @@
     <div class="parallax-content baner-content" id="home">
         <div class="container">
             <div class="first-content">
-                <h1>GitOps@Baloise</h1>
+                <h1>Preview Test</h1>
```
Comment (joachimprinzbach) [inline static/index.html:34]:
```suggestion
                <h1>Preview Preview</h1>
```

Comment (niiku) [reply static/index.html:34]: thanks for.. that...

**Pull request closed without merging**